worlds/ahit/DeathWishRules.py

```diff
@@ -343,6 +346,7 @@ def create_enemy_events(world: "HatInTimeWorld"):
 
 def set_enemy_rules(world: "HatInTimeWorld"):
     no_tourist = "Camera Tourist" in world.excluded_dws or "Camera Tourist" in world.excluded_bonuses
+    difficulty: Difficulty = Difficulty(world.options.LogicDifficulty)
```

Comment:
Could this be used instead?
```suggestion
    difficulty: Difficulty = get_difficulty(world)
```